sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
+
+Because this is just a normal route in controller, there's really nothing special about it. It's not automatically secured and there's no security being applied to this. I mean, check it out: I'm not even logged in right now, and I am successfully on my admin dashboard.
+
+So let's secure it! I'll also do this with annotations. I already have SensioFrameworkExtraBundle installed. So I can say `#[IsGranted()]` and hit "tab" to auto complete that. Now we'll look for `ROLE_ADMIN`. That's kind of my base admin role, so all admin users will have that role. Now when I refresh... Beautiful! We bounced back over to the login page!
+
+So to log in, if you open `src/DataFixtures/AppFixtures`, I have a bunch of dummy users in the database. There is a super admin, a normal admin, and then somebody known as a moderator. We'll talk more about those later, when we get deeper into how to secure different parts of your admin with different roles.
+
+I'll log in with "admin@example.com"... password "admin password"...
+
+And... beautiful! Now we're back to our dashboard!
+
+Of course, if you want to, instead of securing `IsGranted` with the PHP attribute, you can use `$this->denyAccessUnlessGranted()`. You can also go to `config/packages/security.yaml` and, down at the bottom, you can always just use access control for this - whichever one you like better.
+
+So, our dashboard is the jumping off point for our admin, but there's nothing too special here. This page has a title, some menu items, and a nice little user menu over here. Eventually, we'll be able to actually generate something cool on this page, like links or graphs or something. By the way, styling is all done in Bootstrap 5 with FontAwesome.
+
+So let's see if we can customize the dashboard a little bit. One of the best things about EasyAdmin is that all the config is done in PHP, usually via methods in your controller. So, for example, want to configure the dashboard? There is a configured dashboard method for that!
+
+We can change the title of the page to call during overflow admin. And I wanna refresh, we get Coran overflow admin, and there are a number of other methods. You can just look at the auto complete here. So, related to the fab icon path, there's various ways that you can modify things. You can see there's things about the sidebar being minimized. That's referring to a nice little feature here where you can double click on this little thing to expand or collapse the sidebar.
+
+The main part of the dashboard is really the menu items over here, which we only have one of right now. It's controlled by `ConfigureMenuItems`. So, just to show that we can, let's change this FA to `fa-dashboard.` This leverages the FontAwesome library. So `fa-dashboard` corresponds to that little icon right there.
```

Comment:
`configureMenuItems()`

`()` because it's a function (the way to know this is that, in the video, after `configureMenuItems`, you will see a `(`... and eventually a `)`.